packages/run-protocol/src/vpool-xyk-amm/pool.js

```diff
@@ -301,15 +301,13 @@ const finish = context => {
       ),
     );
 
-  const notifier = makeNotifierFromSubscriber(subscriber);
-
   const toCentralPriceAuthority = makePriceAuthority(
     getInputPriceForPA,
     getOutputPriceForPA,
     secondaryBrand,
     centralBrand,
     timer,
-    notifier,
+    subscriber,
```

Comment:
I'm not certain I understand this change. It doesn't seem like a pure refactor; it seems to have observable changes. Are they testested? Or perhaps the change isn't visible from any exported API?

Reply:
This one is a pure refactor. We shouldn't need any Notifier within the contract code now that we have prefix-lossy Subscription.